catchup/catchpointService.go

```diff
@@ -335,12 +339,17 @@ func (cs *CatchpointCatchupService) processStageLedgerDownload() (err error) {
 	return nil
 }
 
-// updateVerifiedAccounts update the user's statistics for the given verified accounts
-func (cs *CatchpointCatchupService) updateVerifiedAccounts(addedTrieHashes uint64) {
+// updateVerifiedCounts update the user's statistics for the given verified hashes
+func (cs *CatchpointCatchupService) updateVerifiedCounts(kvCount, accountCount uint64) {
```

Comment:
nit: Swap the args? Since we always do these things in the order: account stuff, then kv stuff.

Reply:
@jannotti I swapped the args in https://github.com/algorand/go-algorand/pull/4808/commits/015199a283e0114df600ac77252b1c25ed1c53c8.  Can you have a look and resolve if it looks good?